HW08/ArrayDataException.java

```diff
@@ -0,0 +1,6 @@
+public class ArrayDataException extends NumberFormatException {
```

Comment:
Not a good idea to extend exiting specific exceptions. The better approach is to create your own exception from the Exception class.